tests/fields/test_float.py

```diff
@@ -13,9 +13,11 @@ def test_float_field():
     form = F(DummyPostData(a=["v"], b=["-15.0"]))
```

Comment:
Need to test the "step" values association to the float input data. So a step of "any" is good with "1.234" but a step of "0.1" should not validate a "1.234" input.

Reply:
thanks for your input, it looks like this related to this issue #658.